drivers/hmis/spec/requests/hmis/form_definition_lookup_spec.rb

```diff
@@ -135,6 +135,44 @@
       end
     end
   end
+
+  describe 'Form definition project matches' do
+    let(:query) do
+      <<~GRAPHQL
+        query GetFormProjectMatches($id: ID!) {
+           formDefinition(id: $id) {
+            id
+            cacheKey
+            projectMatches {
+              id
+              projectName
+              organizationName
+              dataCollectedAbout
+            }
+          }
+        }
+      GRAPHQL
+    end
+
+    let!(:o1) { create :hmis_hud_organization, data_source: ds1 }
+    let!(:p1) { create :hmis_hud_project, data_source: ds1, organization: o1 }
+    let!(:p2) { create :hmis_hud_project, data_source: ds1, organization: o1 }
+
+    let!(:form) { create :hmis_form_definition, identifier: 'form-def' }
+    let!(:all_clients) { create :hmis_form_instance, definition_identifier: 'form-def', entity: o1, active: true, data_collected_about: 'ALL_CLIENTS' }
+    let!(:hoh_and_adults) { create :hmis_form_instance, definition_identifier: 'form-def', entity: p1, active: true, data_collected_about: 'HOH_AND_ADULTS' }
```

Comment:
this factory does accept a definition, so it might be cleaner to just use that. The identifier itself is not really relevant to this test

```
    let!(:form) { create :hmis_form_definition }
    let!(:all_clients) { create :hmis_form_instance, definition: form, entity: o1, active: true, data_collected_about: 'ALL_CLIENTS' }
    let!(:hoh_and_adults) { create :hmis_form_instance, definition: form, entity: p1, active: true, data_collected_about: 'HOH_AND_ADULTS' }
``